### PR TITLE
Add OHC Notebooks

### LINKED ISSUE #26: Create OHC notebook#1 - opening datasets in Xarray.
How to import modules?
What is in a dataset?
What is the difference between a datset and a dataarray?

### PR DESCRIPTION
**Pull request by jukent**
Closes #26

### CODE REVIEW

**Comment by jukent** [comment]
@andersy005 Sorry I left out one sub-directory. Will fix that soon.

I left out the explicit dask stuff, should I skip chunking too?

I will think about visualizations,, or at least formatting the text better.

**Comment by andersy005** [comment]
> I left out the explicit dask stuff, should I skip chunking too?

Yes... Chunking could be introduced in another notebook, let's say if you were using `xr.open_mfdataset()` for instance.

**Comment by andersy005** [comment]
@jukent, let me know when this is ready and I will take another look. 

I saw the following line

```ds = xr.open_dataset(file, chunks = {'time': 16})```

in the second notebook, and I wasn't sure whether this is ready.

File: notebooks/Welcome.ipynb [hunk 1]
```diff
@@ -230,6 +230,22 @@
     "\n",
     "\n"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "  <p>Next: <a href=\"01_modules_and_xarray_datasets.ipynb\">Sample Workflow: Calculate Ocean Heat Content</a></p>\n",
```
Comment (andersy005) [inline notebooks/Welcome.ipynb]:
```html
<p>Next: <a href="01_modules_and_xarray_datasets.ipynb">Sample Workflow: Calculate Ocean Heat Content</a></p>
```

----> 

```html
<p>Next: <a href="./sample_workflows/ocean_heat_content/01_modules_and_xarray_datasets.ipynb">Sample Workflow: Calculate Ocean Heat Content</a></p>
```

**Comment by jukent** [comment]
Everything should be set except for 
- the table of contents (@andersy005 )
- potentially hiding my solutions

File: notebooks/sample_workflows/ocean_heat_content/01_modules_and_xarray_datasets.ipynb [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+    "## Going further:\n",
+    " - Xarray Documentation: http://xarray.pydata.org/en/stable/index.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "  <p>Previous: <a href=\"01_modules_and_xarray_datasets.ipynb\">What is Git?</a></p>\n",
```
Comment (andersy005) [inline notebooks/sample_workflows/ocean_heat_content/01_modules_and_xarray_datasets.ipynb]: We have dead links here for both `next` and `previous`

File: notebooks/sample_workflows/ocean_heat_content/02_subselecting_and_indexing_data.ipynb [hunk 3]
```diff
@@ -0,0 +1,548 @@
...
+    "- Xarray's Indexing and Selecting Data Documentation: http://xarray.pydata.org/en/stable/indexing.html"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-success\">\n",
+    "  <p>Previous: <a href=\"01_modules_and_xarray_datasets.ipynb\">Modules and Xarray Datasets</a></p>\n",
+    "  <p>Next: <a href=\"03_selecting_and_indexing_data.ipynb\">Selecting and Indexing Data</a></p>\n",
```
Comment (andersy005) [inline notebooks/sample_workflows/ocean_heat_content/02_subselecting_and_indexing_data.ipynb]: This link should point to the `03_units.ipynb` notebook, right?

**Comment by jukent** [comment]
Either you can fix the table of contents or I will ask you how to tomorrow.

**Comment by andersy005** [comment]
@jukent, 

I fixed the TOC issues, added the solutions into `solutions/` folder in conjunction with the `%load solutions/solution_....py` magic. 

- I didn't add solutions for tasks that didn't have answers in the notebooks. You may need to add those. 

- I created a `00_intro.ipynb` notebook to provide a short intro prior diving into the other 4 notebooks.

- If you need to add changes, remember to run `git pull` since I pushed directly to your master branch.

**Comment by jukent** [comment]
I'll add those solutions tomorrow! Thanks!

**Comment by jukent** [comment]
Added solutions!

**Pull request merged**